app/src/main/java/com/humberto/tasky/core/data/auth/AccessTokenAuthenticator.kt

```diff
@@ -50,7 +49,8 @@ class AccessTokenAuthenticator @Inject constructor(
             result = AuthInfo(
                 accessToken = accessTokenResponse.accessToken,
                 refreshToken = authInfo?.refreshToken ?: "",
-                userId = authInfo?.userId ?: ""
+                userId = authInfo?.userId ?: "",
+                fullName = authInfo?.fullName ?: ""
```

Comment:
Why can the properties of `AuthInfo` be null? If there is a non-null AuthInfo object, they certainly exist. Giving a data class nullable fields that always exist for a non-null class instance is an anti-pattern, since it communicates the wrong intention (that these properties are optional).
To indicate that there might or might not be info about the authenticated user, you'd make the whole object nullable instead (which you did here), but after a null-check you could then safely access the mandatory arguments of that object.